dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/DataSourceController.java

```diff
@@ -342,27 +343,43 @@ public Result getKerberosStartupState(@Parameter(hidden = true) @RequestAttribut
 
     @Operation(summary = "tables", description = "GET_DATASOURCE_TABLES_NOTES")
     @Parameters({
-            @Parameter(name = "datasourceId", description = "DATA_SOURCE_ID", required = true, schema = @Schema(implementation = int.class, example = "1"))
+            @Parameter(name = "datasourceId", description = "DATA_SOURCE_ID", required = true, schema = @Schema(implementation = int.class, example = "1")),
```

Comment:
could you add this change to `incompatible.md`? because we change the interface of public method

Reply:
Done